platform-operator/platformcontroller.go

```diff
@@ -603,57 +594,50 @@ func deleteResourcePolicy(resPolicySpec interface{}, namespace string) {
 
 func flatten(yaml_contents map[string]interface{}, types_dict map[string]apiextensionsv1beta1.JSONSchemaProps) map[string]apiextensionsv1beta1.JSONSchemaProps {
```

Comment:
Can you verify if flatten method is being referred to anywhere?
If not, we can delete it.

Reply:
sure

Reply:
We are using it in the getChartValueTypes function

Reply:
But getChartValueTypes() is part of the commented out code that you have removed.
So I think we can safely remove flatten().